components-api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -38,6 +38,8 @@ object ProcessCompilationError {
 
   }
 
+  sealed trait GlobalError { self: ProcessCompilationError => }
```

Comment:
1. Let's call it `GlobalScenarioGraphError` or `ScenarioGraphLevelError` or sth similar. Global/local naming is always confusing. You never now when you'll have sth that need to be "even more global" or "even more local"
2. Let's add a comment that these errors should be thrown when the error can't be fixed by node parameters editing. We also treat otherwise node ids.

Reply:
Done :+1: